templates/show_params.mako

```diff
@@ -174,11 +174,26 @@
         %if job:
             <tr><td>Tool Exit Code:</td><td>${ job.exit_code | h }</td></tr>
         %endif
-        <tr><td>History Content API ID:</td><td>${encoded_hda_id}</td></tr>
+        <tr><td>History Content API ID:</td>
```

Comment:
Should we also change the name to indicate that it is a decoded id for the admin?

Reply:
I thought this is clear for people that are actually using IDs for anything. What would you suggest?

Reply:
Simply adding `(decoded)`? If this is obvious, all good.

Reply:
I had it there, but @natefoo called it superfluous.

Reply:
ok :)